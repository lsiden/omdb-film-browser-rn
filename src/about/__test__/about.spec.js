import React from "react"
import renderer from "react-native-test-utils"

import { about as About } from "../"

const defaultProps = () => {
  return {
    dispatchToPrev: jest.fn(),
  }
}

const createWrapper = (props = {}) =>
  renderer(<About {...{ ...defaultProps(), ...props }} />)

it("renders", () => {
  const wrapper = createWrapper()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
