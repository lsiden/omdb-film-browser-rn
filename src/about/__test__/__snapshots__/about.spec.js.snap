// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<RCTScrollView
  style={
    Object {
      "alignContent": "flex-start",
      "backgroundColor": "#05347f",
      "height": "99%",
      "marginTop": 5,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginTop": 20,
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "marginLeft": 5,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          onPress={undefined}
          style={
            Array [
              false,
              false,
              false,
              Object {
                "alignItems": "center",
                "backgroundColor": "transparent",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
          underlayColor="white"
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 24,
                },
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  undefined,
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            î—„
          </Text>
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 18,
            "marginBottom": 10,
          }
        }
      >
        About
      </Text>
      <View
        style={
          Object {
            "width": 24,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "transparent",
            }
          }
        >
          x
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": 15,
          "marginTop": 15,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 14,
            "marginBottom": 10,
          }
        }
      >
        Written By:
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
          }
        }
      >
        Lawrence Siden
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Westside Consulting LLC
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        westsideconsultingllc@gmail.com
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": 15,
          "marginTop": 15,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 14,
            "marginBottom": 10,
          }
        }
      >
        Credits:
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Create React Native App
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Open Movie Database
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        Icons8
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "marginBottom": 15,
          "marginTop": 15,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 14,
            "marginBottom": 10,
          }
        }
      >
        Source code:
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "white",
            "fontSize": 16,
            "marginBottom": 10,
            "textDecorationLine": "underline",
          }
        }
      >
        react-native-omdb-viewer
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
